src/components/results/next-queries-group.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="x-list x-padding--06 x-background--neutral-95 x-list--gap-06">
+    <div class="x-list x-list--gap-03">
+      <h1 class="x-title2 x-text--bold">{{ $t('nextQueriesGroup.title') }}</h1>
+      <p
+        v-html="$t('nextQueriesGroup.message', { query: $x.query.search })"
+        class="x-text x-font-size--05"
+      />
+    </div>
+    <NextQueries
+      #suggestion="{ suggestion }"
+      :suggestions="nextQueries"
+      class="x-list--gap-06"
+      :max-items-to-render="3"
+    >
+      <!-- eslint-disable max-len-->
+      <NextQuery
+        #default="{ suggestion: nextQuery }"
+        :suggestion="suggestion"
+        class="x-tag x-tag--pill x-list x-list--gap-03 x-padding--bottom-03 x-padding--top-03 x-border-color--neutral-10 x-background--neutral-100"
+      >
+        <LightBulbOn class="x-icon--l" />
+        <span class="x-font-size--05 x-ellipsis">{{ nextQuery.query }}</span>
```

Comment:
```suggestion
        <span class="x-font-size--05 x-ellipsis x-flex-auto">{{ nextQuery.query }}</span>
```

We need it to expand add space between the items inside the next query.


![image](https://user-images.githubusercontent.com/6597815/150958442-162f442a-9ac0-47bb-ba39-c5c57831b236.png)